### PR TITLE
Allow specifying different branch name to compare with

### LINKED ISSUE #10: specify different branch to compare with latest release
`master` is neat, but perhaps you want something different

### PR DESCRIPTION
**Pull request by asizikov**
Fixes #10

* Added `/branch` parameter with `master` as a default value
* Some unreachable code removed (for list `| xs::xss` and `| []` patterns is enough)

### CODE REVIEW

File: ReleaseNotesGenerator/ReleaseNotesGenerator.fsproj [hunk 1]
```diff
@@ -25,7 +25,7 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <DocumentationFile>bin\Debug\ReleaseNotesGenerator.XML</DocumentationFile>
     <Prefer32Bit>true</Prefer32Bit>
-    <StartArguments>/repo octokit/octokit.net</StartArguments>
+    <StartArguments>/repo octokit/octokit.net /branch master</StartArguments>
```
Comment (shiftkey) [inline ReleaseNotesGenerator/ReleaseNotesGenerator.fsproj]: Do we need to specify this when debugging?

Comment (asizikov) [inline ReleaseNotesGenerator/ReleaseNotesGenerator.fsproj]: Not really, it's a default value. I just followed the pattern.

**Comment by asizikov** [comment]
Review suggestions were implemented.

**Comment by shiftkey** [comment]
@asizikov thanks!

**Comment by asizikov** [comment]
Hell yeah, my first F# PR :tada:

**Pull request merged**